clients/go/apis/v1alpha1/config.go

```diff
@@ -17,7 +17,7 @@ package v1alpha1
 import (
 	"fmt"
 
-	"go.uber.org/multierr"
+	"github.com/hashicorp/go-multierror"
```

Comment:
Somewhere I saw a comment about check nil err. I think the old package can do that as well. But it's good we standardize on one lib for multi error handling.

Reply:
Can you add it to our running doc?

Reply:
Already did!